src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Command/Entity/ChooseEntityTypeCommand.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Copyright 2018 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\ServiceProviderDashboard\Infrastructure\DashboardBundle\Command\Entity;
+
+use Surfnet\ServiceProviderDashboard\Domain\Entity\Entity;
+
+class ChooseEntityTypeCommand
+{
+    /**
+     * @var string
+     */
+    private $type;
+
+    public function __construct($type = Entity::TYPE_SAML)
```

Comment:
Why wouldn't you omit the default?

Reply:
This is pure convenience. This way the SAML option is checked by default without the need for setting a default in the controller. What would you suggest otherwise?